lib/application/README.md

```diff
@@ -49,6 +69,17 @@ APP.exampleApp.render({ productId : 1 });
 
 ## Methods to Be Called by Core Application Code
 
+- `processReady(error)`: A method that your core application code can call in
+  order to access calls to `ready` that have occurred. The only parameter it
+  accepts is an optional one, for an error to be passed to the callbacks in
+  its queue, and to reject its Promise with, if used in an environment that
+  supports ES6 Promises. It will then replace the original definition of the
+  `ready` method. Queued items are processed synchronously. If your
+  application is being loaded via the BV Loader, the Loader will handle
```

Comment:
Maybe a bit pedantic, but we shouldn't be creating references to BV Loader or other internal BV things in this repo, which is OSS.